src/sentry/workflow_engine/processors/action.py

```diff
@@ -180,3 +181,29 @@ def get_notification_plugins_for_org(organization: Organization) -> list[PluginS
             plugin_map[plugin.slug] = PluginService(plugin)
 
     return list(plugin_map.values())
+
+
+def get_integration_services(organization_id: int) -> dict[int, list[tuple[int, str]]]:
+    """
+    Get all Pagerduty services and OpsGenie teams for an organization's integrations.
+    """
+
+    org_ints = integration_service.get_organization_integrations(
+        organization_id=organization_id, providers=["opsgenie", "pagerduty"]
```

Comment:
nit: use `IntegrationProviderSlug` from `sentry.integration.types`
```suggestion
        organization_id=organization_id, providers=[IntegrationProviderSlug.OPSGENIE, IntegrationProviderSlug.PAGERDUTY]
```

Reply:
facts we have too many different ones 😢